src/pages/workspace/companyCards/addNew/CardTypeStep.tsx

```diff
@@ -0,0 +1,121 @@
+import React, {useMemo, useState} from 'react';
+import type {ValueOf} from 'type-fest';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import * as CompanyCards from '@userActions/CompanyCards';
+import CONST from '@src/CONST';
+
+function CardTypeStep() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const [typeSelected, setTypeSelected] = useState<ValueOf<typeof CONST.COMPANY_CARDS.CARD_TYPE>>();
+
+    const submit = () => {
+        CompanyCards.setAddNewCompanyCardStepAndData({
+            step: CONST.COMPANY_CARDS.STEP.CARD_INSTRUCTIONS,
+            data: {
+                cardType: typeSelected,
+            },
+            isEditing: false,
+        });
+    };
+
+    const handleBackButtonPress = () => {
+        CompanyCards.setAddNewCompanyCardStepAndData({step: CONST.COMPANY_CARDS.STEP.CARD_TYPE});
+    };
+
+    const data = useMemo(() => {
+        const options = [];
+        options.push(
```

Comment:
```suggestion
    const data = useMemo(() => [
        Objects
    ]
```
Can be directly return the array in case we want to avoid storing in separate array variable and push.